lib/modules/platform/types.ts

```diff
@@ -249,7 +249,14 @@ export interface Platform {
   ensureIssue(
     issueConfig: EnsureIssueConfig,
   ): Promise<EnsureIssueResult | null>;
-  massageMarkdown(prBody: string): string;
+  massageMarkdown(
+    prBody: string,
+    /**
+     * Useful for suggesting the use of rebase label when there is no better
+     * way, e.g. for Gerrit.
+     */
+    rebaseLabel?: string,
```

Comment:
we should make this a config object for easier extending later. 


maybe pass full `RenovateConfig` object 🤔

@rarkins @secustor wdyt?

Reply:
In that case, feel free to merge this PR, and I will send a follow-up making Gerrit leverage it.